packages/editor/src/components/collab-sidebar/index.js

```diff
@@ -117,7 +117,8 @@ export default function CollabSidebar() {
 		commentLastUpdated,
 	} = useBlockComments( postId );
 	useEnableFloatingSidebar(
-		unresolvedSortedThreads.length > 0 || showCommentBoard
+		isLargeViewport &&
+			( unresolvedSortedThreads.length > 0 || showCommentBoard )
```

Comment:
There's no point in enabling the `useEnableFloatingSidebar` hook logic on smaller screens.